editor/src/clj/editor/asset_browser.clj

```diff
@@ -691,6 +691,9 @@
 (defn- drag-detected [^MouseEvent e selection]
   (let [resources (roots selection)
         files (fileify-resources! resources)
+        paths (->> resources
+                   (mapv resource/proj-path)
+                   (string/join "\n"))
```

Comment:
By adding plain paths to content we avoid unnecessary back and forth (see removed `get-image-resources-from-files`). We also allow dropping files from builtin assets that wasn't previously working because `fileify-resources!` creates a temporary path. What we really want on drop is a string that will be resolved to an existing workspace resource.